.github/workflows/azure_kubernetes_aks_single_region_tests.yml

```diff
@@ -0,0 +1,593 @@
+                    echo "CLUSTER_NAME=$RESOURCE_PREFIX-aks"
+                    echo "RESOURCE_GROUP_NAME=$RESOURCE_PREFIX-rg"
+                  } >> "$GITHUB_ENV"
+
+            - name: Export S3_BACKEND_BUCKET
+              id: s3_prefix
+              run: |
+                  set -euo pipefail
+                  echo "S3_BACKEND_BUCKET_PREFIX=azure/kubernetes/azure-kubernetes-aks-single-region/" | tee -a "$GITHUB_OUTPUT"
+
+            - name: Create K8S cluster and login
+              uses: ./.github/actions/azure-kubernetes-aks-single-region-create
+              id: create_cluster
+              # Do not interrupt tests; otherwise, the Terraform state may become inconsistent.
+              if: always() && success()
+              with:
+                  resource-prefix: ${{ env.RESOURCE_PREFIX }}
+                  cluster-name: ${{ env.CLUSTER_NAME }}
+                  resource-group-name: ${{ env.RESOURCE_GROUP_NAME }}
+                  s3-backend-bucket: ${{ env.S3_BACKEND_BUCKET }}
+                  s3-bucket-region: ${{ env.S3_BUCKET_REGION }}
+                  s3-bucket-key-prefix: ${{ steps.s3_prefix.outputs.S3_BACKEND_BUCKET_PREFIX }}${{ steps.camunda-version.outputs.CAMUNDA_VERSION }}/
+                  tf-modules-revision: ${{ github.ref }}
+                  tf-modules-path: ${{ env.TF_MODULES_PATH }}
+                  ref-arch: aks-single-region
+
+            - name: Source env from Terraform
+              id: db_env
+              working-directory: ${{ env.TF_MODULES_PATH }}/azure/kubernetes/aks-single-region/
+              run: |
+                  set -euo pipefail
+
+                  # Source the shell script that exports your DB-related variables
+                  # shellcheck disable=SC2086
+                  source $GITHUB_WORKSPACE/azure/kubernetes/aks-single-region/procedure/vars-create-db.sh
+
+                  {
+                    echo "POSTGRES_FQDN=$POSTGRES_FQDN"
+                    echo "POSTGRES_PORT=$POSTGRES_PORT"
+                    echo "POSTGRES_ADMIN_USERNAME=$POSTGRES_ADMIN_USERNAME"
+                    echo "POSTGRES_ADMIN_PASSWORD=$POSTGRES_ADMIN_PASSWORD"
+                    echo "DB_KEYCLOAK_NAME=$DB_KEYCLOAK_NAME"
+                    echo "DB_KEYCLOAK_USERNAME=$DB_KEYCLOAK_USERNAME"
+                    echo "DB_KEYCLOAK_PASSWORD=$DB_KEYCLOAK_PASSWORD"
+                    echo "DB_IDENTITY_NAME=$DB_IDENTITY_NAME"
+                    echo "DB_IDENTITY_USERNAME=$DB_IDENTITY_USERNAME"
+                    echo "DB_IDENTITY_PASSWORD=$DB_IDENTITY_PASSWORD"
+                    echo "DB_WEBMODELER_NAME=$DB_WEBMODELER_NAME"
+                    echo "DB_WEBMODELER_USERNAME=$DB_WEBMODELER_USERNAME"
+                    echo "DB_WEBMODELER_PASSWORD=$DB_WEBMODELER_PASSWORD"
+                    echo "RESOURCE_GROUP_NAME=$RESOURCE_GROUP_NAME"
+                    echo "CLUSTER_NAME=$CLUSTER_NAME"
+                    echo "RESOURCE_PREFIX=$RESOURCE_PREFIX"
```

Comment:
if you don't need RESOURCE_PREFIX in any future steps, you could save yourself the exporting.
At least I didn't see it anywhere else in active use.